snippets/newsletter-form.liquid

```diff
@@ -1,5 +1,8 @@
 
-<form action="{{ settings.mailing_list_form_action }}" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank" class="input-group">
+{% capture action_link %}
+{% if settings.mailing_list_form_action != "" %}{{settings.mailing_list_form_action}}{%else%}javascript:void(0); {% endif %}
```

Comment:
Is `!= ""` working?
Liquid has {% if my_variable == blank %}, do not know if previuos is valid.

Reply:
Yes. It's work fine.